code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -24,6 +24,8 @@
 
 	var/power_efficiency = 1.0
 
+	mob_size = 16
+
```

Comment:
`mob_size` should be define
One of this or new one:
```
// Defines mob sizes, used by lockers and to determine what is considered a small sized mob, etc.
#define MOB_LARGE  		40
#define MOB_MEDIUM 		20
#define MOB_SMALL 		10
#define MOB_TINY 		5
#define MOB_MINISCULE	1
```

Reply:
holy fuck what's with those numbers, THAT is why drones are instagibbing.

Who chose these values and how?